arangod/MMFiles/MMFilesEngine.cpp

```diff
@@ -3424,23 +3424,7 @@ Result MMFilesEngine::createLoggerState(TRI_vocbase_t* vocbase, VPackBuilder& bu
   // "clients" part
   builder.add("clients", VPackValue(VPackValueType::Array));  // open
   if (vocbase != nullptr) {                                   // add clients
-    auto allClients = vocbase->getReplicationClients();
```

Comment:
is this shown in the UI somewhere ?

Reply:
Yes, the web UI shows the connected clients in the "REPLICATION" tab on a master